WordPress/src/main/java/org/wordpress/android/ui/accounts/signup/SignupEpilogueFragment.java

```
@@ -836,18 +839,15 @@ public void run() {
                         new GravatarApi.GravatarUploadListener() {
                             @Override
                             public void onSuccess() {
-                                // FIXME: log analytics
                                 AppLog.i(T.NUX, "Google avatar download and Gravatar upload succeeded.");
-                                AnalyticsTracker.track(AnalyticsTracker.Stat.ME_GRAVATAR_UPLOAD_UNSUCCESSFUL);
+                                AnalyticsTracker.track(Stat.ME_GRAVATAR_UPLOADED);
                             }
 
                             @Override
-                            public void onError(String exceptionClass, String exceptionMessage) {
+                            public void onError(@NonNull ErrorType errorType) {
                                 AppLog.i(T.NUX, "Google avatar download and Gravatar upload failed.");
-                                // FIXME: Don't track exceptions caused by poor internet connectivity
```

Previously some exceptions were ignored. I wasn't sure what to do, this could actually hide some underlying problems like a too short default timeout, so I decided to drop the filter.

---

It looks better this way. We don't want to hide any exceptions, at least in the app.